plugins/filters/askAI/reference.md

```diff
@@ -11,6 +11,7 @@ A gitStream plugin to interact with AI models. Currently works with `ChatGPR-4o-
 | Param   | Type     | Description                                     |
 | ------- | -------- | ----------------------------------------------- |
 | context | `Object` | The context that will be attached to the prompt |
+| role    | `string` | Role instructions for the conversationThe       |
```

Comment:
should say if case sensitive or not, if free text or not (if not, what the possible values), and prob that it will sent to the agent as its role